app/views/partials/_navbar.html.erb

```diff
@@ -1,41 +1,45 @@
-<div id="nav-bar">
-  <div class="top-left section">
+<div id="nav-bar" class="px-2">
     <a
       href="<%= root_path %>"
       title="OpenFlightHPC Carbon Leaderboard"
+      class="top-left section"
     >
       <%= image_tag "logo.svg", class: "logo" %>
+      <div id ="logo-text">
+        <span>OpenFlightHPC</span><br>
+        <span>Carbon leaderboard</span>
+      </div>
     </a>
-    <div>
-      <span>OpenFlightHPC</span><br>
-      <span>Carbon leaderboard</span>
-    </div>
-  </div>
 
   <div class="centre section">
-      <div class="dropdown">
-        <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false">
-          Leaderboards
-        </button>
-        <ul class="dropdown-menu">
-          <%= link_to "Server leaderboard", leaderboard_path, class: "dropdown-item" %>
-        </ul>
+      <div id="leaderboard-link-wrapper" class="nav-link-wrapper nav-link-dropdown-wrapper d-flex align-items-center">
+        <%= link_to "Leaderboards", leaderboard_path, class: "nav-link px-3" %>
+        <div class="dropdown-wrapper d-flex justify-content-center">
+          <ul class="dropdown-list">
+            <%= link_to "Device leaderboard", leaderboard_path, class: "nav-dropdown-link px-2 glaring" %>
+            <%= link_to "Server leaderboard", leaderboard_path, class: "nav-dropdown-link last-item px-2 glaring" %>
+          </ul>
+        </div>
+      </div>
+      <div id="leaderboard-link-wrapper" class="nav-link-wrapper d-flex align-items-center">
+        <%= link_to "Servers", show_devices_path, class: "nav-link px-3" %>
       </div>
-      <%= button_to "Servers", show_devices_path, method: :get, class: 'btn' %>
   </div>
 
   <div class="top-right section">
     <% if user_signed_in? %>
-      <div class="dropdown account">
-        <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false">
-          <%= current_user.username %>
-        </button>
-        <ul class="dropdown-menu">
-          <%= button_to "Sign out", destroy_user_session_path, method: :delete, class: "btn dropdown-item" %>
-        </ul>
+      <div id="leaderboard-link-wrapper" class="nav-link-wrapper nav-link-dropdown-wrapper d-flex align-items-center">
+        <%= link_to "#{['Hi', 'Hello', 'Welcome'].sample}, #{current_user.username}", "#", class: "nav-link px-3" %>
```

Comment:
Love the changing greeting :D 

Why is this a link?

Reply:
The link is a preparation for a possible profile page in the future.

Reply:
I can confirm it is helpful for this to be a link, since I came to this PR mainly to ask that a profile page link be added to the navbar! Technically one already exists in this version under `/user/<username>` although it's just a table of records owned by that user (same page that gives your auth token). In v0.0.2 with my latest PR, this page gets changed a little and it'd be helpful to always have a link to it.